Makefile

```diff
@@ -54,3 +53,15 @@ ut-circle: calico_mesos_builder.created dist/calico_isolator
 	--with-xunit --xunit-file=/circle_output/output.xml; RC=$$?;\
 	[[ ! -z "$$COVERALLS_REPO_TOKEN" ]] && coveralls || true; exit $$RC'
 
+rpm:
+	mkdir -p dist
+	chmod 777 `pwd`/dist
+	cd packages && docker build -t calico-mesos-rpm-builder .
+	cd packages && docker run \
+	 -v $(WD)/dist:/opt/rpms \
+	 calico-mesos-rpm-builder bash -c 'rpmbuild -ba /root/calico-mesos.spec && \
+	 cp `find /root/rpmbuild/RPMS -type f` /opt/rpms'
```

Comment:
I think this can be simplified a bit to `cp /root/rpmbuild/RPMS/*.rpm /opt/rpms`? Unless the RPMs are in subfolders within RPMS somewhere